client/app/components/visualizations/editor/withControlLabel.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import hoistNonReactStatics from 'hoist-non-react-statics';
+import * as Grid from 'antd/lib/grid';
+
+import './control-label.less';
+
+// TODO: Control label should reflect disabled state
+// <Typography.Text disabled={disabled}>label text</Typography.Text>
```

Comment:
Perhaps create an issue on GH instead of that? (unless you intend to do it in this PR) I think those comments are more easily lost than the issues :confused:

Reply:
I forgot about this :man_facepalming: Now implemented in this PR https://github.com/getredash/redash/pull/4382/commits/7cad47782985c9f49258f77753453bd6c5780755